doc/kernel/services/polling.rst

```diff
@@ -185,15 +185,20 @@ to :c:macro:`K_POLL_STATE_NOT_READY` by the user.
     {
         for(;;) {
             rc = k_poll(events, ARRAY_SIZE(events), K_FOREVER);
+            // NOTE: do not use an if-else-chain, because multiple events
+            // may be active simultaneously
```

Comment:
Suggest removing this comment, or reword to be less prescriptive.  The existing code isn't actually incorrect, at worst it's inefficient as it requires a separate trip through k_poll() for each handled event (but of course in practice truly simultaneous wakeups are likely to be rare, or else reflect existing app inefficiencies with the "event model" or whatever).

Reply:
Ah, I think, I start to understand the argument. Are you saying, that there will be a round-trip to `k_poll`, because the underlying object is still ready (semaphore given, data available in queue, signal pending, ...)? That makes sense.

Of course it requires, that the app only for example resets the poll-signal in the case it handled that event, which is kind of obvious.

Will remove the comment.